_hoh_071_enemies/mimic.md

```diff
@@ -39,4 +39,8 @@ job_specifics:
     difficulty: Medium
   WAR:
     difficulty: Medium
+  VPR:
+    difficulty: Easy
+    notes:
+      - 'Spinning mimics perfectly will make this fight easy as VPR has very high melee damage'
```

Comment:
Difficulty should not be based on spinning. I'm hesitant to include any information about spinning at all without the disclaimer that it is controversial at best.

Reply:
good point, it's technically an exploit but so was LoS.
you decide, i personally think all information should be available if others are using it to their advantage. i wish there was a page on specific mechanics instead!